docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.7
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
+
```

Comment:
Don't quite understand what this is for and why this is needed in the docker file.

Reply:
Usability. If you get into the shell, you won't be able to use `Ctrl + arrows` to move between words, for me it is annoying, that's why I added this.

Reply:
again I think this is on the same level as zstyle - something that is generic shell/system-related and not about `dvc`